src/App.js

```diff
@@ -171,13 +172,15 @@ const App = () => {
       value: 1000000000000000
     }
 
-    const sendTransaction = () =>
+    const sendTransaction = () => {
```

Comment:
This function needs to return the result from the call to `signer.sendTransaction`

Reply:
ahh yeah whoops! Good catch!! Ill make sure my formatter on this machine is doin wonky stuff like this in the future.